deployment/terraform/assets/cluster.tf

```diff
@@ -520,7 +528,7 @@ resource "aws_security_group_rule" "metrics-ssh" {
   from_port         = 22
   to_port           = 22
   protocol          = "tcp"
-  cidr_blocks       = ["0.0.0.0/0"]
+  cidr_blocks       = ["${local.ifconfig_result}/32"]
```

Comment:
I can see this becoming a bit of an annoyance as we tend to share access to grafana panels and this would essentially limit that without some manual intervention. Not sure if we could consider keeping this open.